WooCommerce/Classes/Yosemite/DeauthenticatedState.swift

```diff
@@ -15,7 +26,9 @@ class DeauthenticatedState: StoresManagerState {
     ///
     func willLeave() { }
 
-    /// NO-OP: During deauth method, we're not running any actions.
+    /// During deauth method, we're not handling actions that don't require access token.
```

Comment:
> we're not handling actions that don't require access token

Sorry, I don't understand this. Does this mean that we are only handling actions that require access token? 🤔

Reply:
This is a typo 😅  I clarified the comment in 2d6aef5aac24954d9b29a5e5b1f6b66e38931023.